cli/hzn.go

```diff
@@ -542,6 +542,7 @@ Environment Variables:
 	mmsObjectDownloadId := mmsObjectDownloadCmd.Flag("id", msgPrinter.Sprintf("The id of the object to download data. This flag must be used with -t.")).Short('i').Required().String()
 	mmsObjectDownloadFile := mmsObjectDownloadCmd.Flag("file", msgPrinter.Sprintf("The file that the data of downloaded object is written to. This flag must be used with -f. If omit, will use default file name in format of objectType_objectID and save in current directory")).Short('f').String()
 	mmsObjectDownloadOverwrite := mmsObjectDownloadCmd.Flag("overwrite", msgPrinter.Sprintf("Overwrite the existing file if it exists in the file system.")).Short('O').Bool()
+	mmsObjectDownloadSkipVerifyDigitalSig := mmsObjectDownloadCmd.Flag("skipVerifyDigitalSig", msgPrinter.Sprintf("Indicate no digital signature verification is applied during data downloading.")).Bool()
```

Comment:
Change the flag name to "--noIntegrity". Change the description to "The download command will not perform a data integrity check on the downloaded object data."